assets/js/apps/search/components/Result.jsx

```diff
@@ -127,8 +139,11 @@ function Result({ result }) {
               </li>
               {result.institution_name
               && (
-                <li className="pull-right">
-                  <span className="badge badge-info mr-2" title={`Institution Name: ${result.institution_name}`}>
+                <li className="float-end">
+                  <span
+                    className="badge bg-info me-2"
+                    title={`Institution Name: ${result.institution_name}`}
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 26 spaces but found 20.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
                          title={`Institution Name: ${result.institution_name}`}
```